src/Tizen.NUI/src/public/BaseComponents/AnimatedImageView.cs

```diff
@@ -372,101 +372,76 @@ private int InternalCurrentFrame
-                tMap.Insert(ImageVisualProperty.FrameDelay, frameDelayProperty);
-
-                arrayProperty.Dispose();
-                indexPropertyArray.Dispose();
-                frameDelayProperty.Dispose();
-            }
-            else
-            {
-                PropertyValue urlProperty = new PropertyValue(url);
-                tMap.Insert(ImageVisualProperty.URL, urlProperty);
-                urlProperty.Dispose();
-            }
-
-            PropertyValue batchSizeProperty = new PropertyValue(batchSize);
-            tMap.Insert(ImageVisualProperty.BatchSize, batchSizeProperty);
-            PropertyValue cacheSizeProperty = new PropertyValue(cacheSize);
-            tMap.Insert(ImageVisualProperty.CacheSize, cacheSizeProperty);
-            PropertyValue loopCountProperty = new PropertyValue(loopCount);
-            tMap.Insert(ImageVisualProperty.LoopCount, loopCountProperty);
-            PropertyValue stopBehaviorProperty = new PropertyValue((int)stopBehavior);
-            tMap.Insert(ImageVisualProperty.StopBehavior, stopBehaviorProperty);
-
-            loopCountProperty.Dispose();
-            cacheSizeProperty.Dispose();
-            batchSizeProperty.Dispose();
-            stopBehaviorProperty.Dispose();
-
-            propertyMap = tMap;
-            PropertyValue mapProperty = new PropertyValue(propertyMap);
-            SetProperty(ImageView.Property.IMAGE, mapProperty);
-            mapProperty.Dispose();
-
-            tMap.Dispose();
-            animatiedImage.Dispose();
+            // Sync as current properties
+            UpdateImage();
         }
 
         /// <summary>
-        /// Play animation
+        /// Update animated-image-relative properties synchronously.
+        /// After call this API, All image properties updated.
         /// </summary>
-        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public new void Play()
+        protected override void UpdateImage()
         {
-            SetValues();
-            base.Play();
-        }
+            if(!imagePropertyUpdatedFlag) return;
 
-        /// <summary>
-        /// Pause animation. Currently pause and stop are same
-        /// </summary>
-        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public new void Pause()
-        {
-            SetValues();
-            base.Pause();
+            if (resourceURLs?.Count != 0)
+            {
+                using(PropertyArray indexPropertyArray = new PropertyArray())
+                {
+                    PropertyArray returnedArr = new PropertyArray();
+                    foreach (var iter in resourceURLs)
+                    {
+                        using(PropertyValue index = new PropertyValue(iter))
```

Comment:
Could you remove this warning message? Or you can suppress this.

src/Tizen.NUI/src/public/BaseComponents/AnimatedImageView.cs#L398
Use recommended dispose pattern to ensure that object created by 'new PropertyValue(iter)' is disposed on all paths.

Reply:
Unfortunatly, current code already use 'using' state, so we can assume that it will be Disposed end of this scope.
I don't know why this message exist. So..just ignore.